cpp/src/io/parquet/page_enc.cu

```diff
@@ -1382,6 +1581,26 @@ __device__ void get_extremum(const statistics_val* stats_val,
   }
 }
 
+static __device__ void get_min(const statistics_val* stats_val,
```

Comment:
Similar to my comment just above. If we need to get one "min" or "max" value, let's return it out of the function instead of writing the result to the input pointer.